lightning/src/chain/onchaintx.rs

```diff
@@ -885,9 +885,10 @@ impl<ChannelSigner: EcdsaChannelSigner> OnchainTxHandler<ChannelSigner> {
 				if let Some((claim_id, _)) = self.claimable_outpoints.get(&inp.previous_output) {
 					// If outpoint has claim request pending on it...
 					if let Some(request) = self.pending_claim_requests.get_mut(claim_id) {
-						//... we need to verify equality between transaction outpoints and claim request
-						// outpoints to know if transaction is the original claim or a bumped one issued
-						// by us.
+						//... we need to check if the pending claim was for a subset of the inputs
```

Comment:
nit: "subset of the outputs" ? A transaction spends outputs not inputs in my head. Also more consistent with later language of the comment "retry claiming the remaining outputs."